app/views/foods/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= render "shared/header" %>
+  <section class="container">
+    <div class="">
+      <%= link_to 'foods/new', :remote => true do %>
+        <input type="button" class="btn btn-primary mb-3" value="Add Food">
+      <% end %>
+    </div>
+
+    <div class="table">
+      <table class="table table-bordered">
+      <tr>
+        <th>Food</th>
+        <th>Measurement Unit</th>
+        <th>Unit Price</th>
+        <th>Action</th>
+      </tr>
+
+      <% @foods.each do |food| %>
```

Comment:
Users should be able to see all foods created by other users and their own foods on the `foods index`. The user is not able to see other users' posts.

Reply:
Done with this error this is what it looks like now...
![CaptureXXIII](https://user-images.githubusercontent.com/83566118/175697087-0856245b-ffdb-43f6-a6de-11d6144f55f3.PNG)

Reply:
:+1: Fixed :heavy_check_mark: